roles/sshd/tasks/main.yml

```diff
@@ -8,6 +8,6 @@
 
 - name: ensure ssh started
   service:
-    name: ssh
+    name: "{{ 'ssh' if ansible_os_family == 'Debian' else 'sshd' }}"
```

Comment:
nit: not a blocker, but yea i would prefer to do include_var stuff rather than ifs inline

Reply:
When we add a third distro, we should totally do that. ;)